### ISSUE TITLE
hack version 0.2.5 Yi Dome Guard chinese version can not use app YI

### ISSUE THREAD

**Issue by tummerry** [roleoroleo/yi-hack-Allwinner-v2#572]
### Discussed in https://github.com/roleoroleo/yi-hack-Allwinner-v2/discussions/570

<div type='discussions-op-text'>

<sup>Originally posted by **tummerry** February  2, 2023</sup>

help me 
hack version 0.2.5 can not link app YI Home and YI Life
Is there a way to help me connect?
I change xxCHxxxxxxxx xxUSxxxxxxxx finish
![image](https://user-images.githubusercontent.com/66394776/216232942-76847aee-b5f3-4e4b-aeab-087165c3840b.png)
![image](https://user-images.githubusercontent.com/66394776/216234060-d8543e79-dd7f-4527-8868-67b9fb69d20c.png)
</div>

**Comment by roleoroleo** [comment]
I don't understand.
Did you change the serial number from CH to US?

**Comment by tummerry** [comment]
yes i change CN    HFCNY36SN04IT8191216    to US    HFUSY36SN04IT8191216 succeed.
But unable to connect YI Home app and YI Life.
I have a backup file for you to look at.
![S__32538776](https://user-images.githubusercontent.com/66394776/216484810-ead9b071-4f4e-404b-899a-a037f10f8ab9.jpg)
[backup_CN.ZIP](https://github.c
[backup_US.ZIP](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10574574/backup_US.ZIP)
om/roleoroleo/yi-hack-Allwinner-v2/files/10574571/backup_CN.ZIP)

Please help me

**Comment by roleoroleo** [comment]
Another user told me that after the serial number change the cam would not connect to the app.
Maybe there is a check in the number.
I will study your backup but I don't know if I'm able to fix the problem.

**Comment by roleoroleo** [comment]
Did the cam connect to the cloud before changing serial number?

**Comment by roleoroleo** [comment]
Please, try to change also TNPCHNG-xxxxxx-YYYYY to TNPUSAL-xxxxxx-YYYYYY.

**Comment by tummerry** [comment]
![image](https://user-images.githubusercontent.com/66394776/216860394-6f49cafa-be19-4580-96d9-c43f77ef2b96.png)
I changed and still can't connect YI App

**Comment by roleoroleo** [comment]
The first char is not ok.
![immagine](https://user-images.githubusercontent.com/39277388/217837318-1a13f2d6-7158-4a46-a3ba-c1af79a86ca6.png)
Please, check again your changes.
